server/src/main/java/org/tctalent/server/request/work/experience/UpdateJobExperienceRequest.java

```diff
@@ -53,6 +53,6 @@ public void setCountry(Long countryId) {
     }
 
     public void setDescription(String description) {
-        this.description = HtmlSanitizer.sanitize(description);
+        this.description = description != null ? HtmlSanitizer.sanitize(description) : null;
```

Comment:
It would be much better to fix HtmlSanitizer.sanitize so that it handles null input.